app/src/pages/save-the-date.tsx

```diff
@@ -0,0 +1,341 @@
+                >
+                  <tbody>
+                    {TranslatedSection(props.save_the_date, props.std_cells)}
+                    <tr
+                      style={{
+                        background:
+                          props.save_the_date.title_color || undefined,
+                        color:
+                          props.save_the_date.background_color || undefined,
+                      }}
+                    >
+                      <td>
+                        <center>
+                          <p style={{ marginTop: "1rem" }}>{tt["follow-us"]}</p>
+                          {props.socialLinks
+                            .filter((s) => s != null)
+                            .map((social) => (
+                              <a href={social.link || ""}>
+                                <img
+                                  src={directusImageUrl(social.logo || "")}
+                                  alt={social.media_name || ""}
+                                  style={{
+                                    height: "2rem",
+                                    margin: "0 0.5rem 1rem 0.5rem",
+                                  }}
+                                />
+                              </a>
+                            ))}
+                        </center>
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </center>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      end-tag
+    </div>
+  );
+}
+
+function TranslatedSection(
+  save_the_date: SaveTheDate,
+  cells: SaveTheDateCell[]
+) {
+  const language_code = locale(useRouter());
+  const commissions_cells = cells.filter((cell) => cell.commission);
+  const clic_cells = cells.filter((cell) => cell.commission == null);
+  const translated_std = getTranslation(save_the_date, language_code);
+  const tt = useTranslationTable();
+
+  return (
+    <>
+      <tr>
+        <td>
+          <center>
+            {save_the_date.image ? (
+              <img
+                src={directusImageUrl(save_the_date.image)}
+                alt="CLIC"
+                style={{ width: "100%", height: "200px" }}
+              />
+            ) : (
+              <></>
+            )}
+            <h1
+              style={{
+                color: save_the_date.title_color || undefined,
+                ...TITLE_STYLE,
+              }}
+            >
+              {tt["save-the-date"].toUpperCase()}
+            </h1>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td style={{ padding: "1rem 0 1rem 0" }}>
+          <center>
+            <a
+              href="https://clic.epfl.ch/en-US/save-the-date/"
+              style={{
+                background: save_the_date.button_color || undefined,
+                ...BUTTON_STYLE,
+              }}
+            >
+              {tt["translated-version"]}
+            </a>
```

Comment:
The link should change depending on whether the current page is in French or English